### ISSUE TITLE
Scanner results are inconsistent

### ISSUE THREAD

**Issue by JWBro** [LMS-Community/slimserver#943]
I have a large library with classical music; the album count is 11522. Given the size I have several copies of the complete library stored on different PCs/servers. One workstation running on the latest version W10 Pro and latest slimserver software gives a count of 11522 albums. Another workstation with the same operating system and slimserver (also up to date) gives a count of 11494 albums. Yet another PC withe same operating system and slimserver counts 11516. Please note: in all cases I have completely redscanned the music library. Any idea what causes the difference in album count?
Jan Broenink

**Comment by JWBro** [comment]
Additional notes: the albums are organized in 6 large folders. The size and content of these 6 folders have been thouroughly checked for consistancy (ie # folders, files and size) and are all the same. All PC's/workstation have been checked for potential HDD errors: none.

**Comment by michaelherger** [comment]
Did you double check all settings in Settings/My Music?

If you were technical you could compare the database files (the albums table).

**Comment by JWBro** [comment]
Hello Michael,

You refer to all settings in Settings/My Music ….
Where should I find these settings?

-          In Logitech Media Server Control Panel …. Under library

-          In Logitech Media Server Control Panel Advanced Tab … in Advanced settings (refers to web control)
No problem to check database files but where can I find these?

-          In Program Files (x86)\server\SQL …. Mysql or SQLite

Thanks & best regards
Jan

**Comment by michaelherger** [comment]
It's in the web UI.

**Comment by JWBro** [comment]
It took a while to check this out. The total size of the music library is over 3.5 Tb with some 11500 albums. Of course I have backups .... I have even two backups. After going through all of it the complete mother-version contains some 11540 albums, the two backups that were created thru copying content. After the LMS scan in one case 11 less albums of some 23 less albums. Found the subfolders in one case were missing data and the other content had files not recognized by the scanner. Aka: files were given wong files types or copied in a strange manner. I have now used a program called FreeSync (latest version) to make sure that all the contents of folders/subfolders was going OK. New scan initiated and seems that the issue has been resolved.
